spk/gitea/Makefile

```diff
@@ -1,12 +1,12 @@
 SPK_NAME = gitea
-SPK_VERS = 1.16.3
-SPK_REV = 3
+SPK_VERS = 1.16.4
+SPK_REV = 4
 SPK_ICON = src/$(SPK_NAME).png
 
 MAINTAINER = publicarray
 DESCRIPTION  = Gitea is a community managed lightweight code hosting solution written in Go.
 DISPLAY_NAME = Gitea
-CHANGELOG = "1. Making setting up SSH easier.<br/> 2. Update to v1.16.1.<br/> 3. Update to v1.16.3."
+CHANGELOG = "1. Making setting up SSH easier.<br/> 2. Update to v1.16.1.<br/> 3. Update to v1.16.3.<br/> 4. Update to v1.16.4."
```

Comment:
please avoid cumulative change log.
The change log should describe the changes to the latest published package only.

Reply:
Fixed